test/unit/runtime/Loader.js

```diff
@@ -141,9 +144,9 @@ suite('Loader.js', function() {
 
   test('LoaderModuleFail', function(done) {
     var code =
-        'import * as a from "./resources/test_a.js";\n' +
-        'import * as b from "./resources/test_b.js";\n' +
-        'import * as c from "./resources/test_c.js";\n' +
+        'import * as a from "./test/unit/runtime/resources/test_a.js";\n' +
+        'import * as b from "./test/unit/runtime/resources/test_b.js";\n' +
+        'import * as c from "./rtest/unit/runtime/esources/test_c.js";\n' +
```

Comment:
Done, I just remove the imports. This test was only to verify that we took the reject() path.